installer.sh

```diff
@@ -161,6 +209,8 @@ Basic usage:
 $ source installer.sh   # Creates the installer session environment       (required)
 
 $ --help                # Prints help (this)
+$ select_browser        # Select browser engine (chromium or mswebview2)
+$ set_target            # Set compilation target file
```

Comment:
"set_target" is better placed below "packages_install" in the intuative order of things